tests/e2e-demos/e2e-demo.go

```diff
@@ -115,6 +115,11 @@ var _ = framework.E2ESuiteDescribe(Label("e2e-demo"), func() {
 					Expect(err).NotTo(HaveOccurred())
 					application = appstudioApp
 
+					_, err = fw.IntegrationController.CreateIntegrationTestScenario(application.Name, namespace, integration.BundleURL, integration.InPipelineName)
```

Comment:
I don't think that creating the IntegrationTestScenario is necessary for the needs of the e2e-demo yet - the Integration service automatically promotes (releases/deploys) Applications without integration test scenarios.
We're expecting to make the `CreateIntegrationTestScenario` more complex in the near future, so this would prevent collisions here too.

Reply:
+1, I think we can add that later when we automate Chapter 3 of the demo script